README.md

```diff
@@ -2260,3 +2261,40 @@ ANSWERS.ponyfillCssVariables({
         onFinally: function() {},
 });
 ```
+
+# Performance Metrics
+
+The SDK uses the Performance API, via `window.performance.mark()`, to create performance metrics regarding ANSWERS.init(), vertical search, and universal search. These marks can be viewed through browser developer tools, or programmatically through `window.performance.getEntries()`.
+
+## ANSWERS.init()
+
+1. `'yext.answers.initStart'` called when ANSWERS.init beings
+
```

Comment:
Are the numbers here important? The 1, 2, 3, etc. Can we remove them?

Reply:
those are the order the the marks are executed, feel like I’d rather have them but dont feel too strong either way. is there a reason you dislike them?

Reply:
Oh, I'd keep them then